astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -72,6 +73,9 @@ def test_format_coord_regression(ignore_matplotlibrc, tmpdir):
 """, sep='\n')
 
 
+# Skipping to avoid PY_SSIZE_T_CLEAN warning with Python 3.9 and Matplotlib 3.0,
+# see https://github.com/astropy/astropy/issues/10954
+@pytest.mark.skipif('MATPLOTLIB_LT_31')
```

Comment:
```suggestion
@pytest.mark.skipif('MATPLOTLIB_LT_31' and sys.version_info >= (3, 9))
```
I suspect the mpl30 CI will be running on <= 3.8 for some time (if there is ever going to be an official setup for py39 + mpl30x at all).

Reply:
Good point, added that. Thanks!